test/test-expressroutes.js

```diff
@@ -84,10 +85,11 @@ test('express routes', function (t) {
 
             stack = Array.prototype.slice.call(parent._router.stack, 3);
 
-            t.strictEqual(stack.length, 2, '2 routes added.');
-            t.strictEqual(stack[1].route.path, '/v1/petstore/pets', '/pets added.');
-            t.strictEqual(stack[1].route.stack.length, 2, '/pets has middleware.');
-            t.strictEqual(stack[1].route.stack[0].name, 'm1', '/pets has middleware named m1.');
+            t.strictEqual(stack.length, 3, '3 routes added.');
+            t.strictEqual(stack[1].regexp.toString(), '/^\\/v1\\/petstore\\/pets(?:\\/(?=$))?$/i', 'bleb');
```

Comment:
I would need an explanation on why you validate the regexp and not the route anymore :)